react_native_fundamentals/presentation.html

```diff
@@ -2,43 +2,1137 @@
+        </section>
+
+        <section class="topic-name-slide">
+          <h4>Project structure overview</h4>
+        </section>
+
+        <section class="bullets-and-picture-slide">
+          <h4 class="slide-name">Default structure</h4>
+          <div>
+            <img src="assets/images/default_structure.png" alt="simple" />
+            <ul>
+              <li>App.js: main component.</li>
+              <li>index.js: entry point of your application.</li>
+              <li>android and ios: native code or project configuration.</li>
+              <li>metro.config.js: metro configuration file.</li>
+              <li>package.json: project metadata, dependancies, and scripts.</li>
+            </ul>
+          </div>
+          <aside class="notes"></aside>
+        </section>
+
+        <section class="text-and-frame-slide">
+          <h4 class="slide-name">Scripts</h4>
+          <div>
+            <p>Runs your android app:</p>
+            <div class="frame">
+              <code class="js" data-trim data-line-numbers>
+                "android": "react-native run-android” 
+              </code>
+            </div>
+
+            <p>Runs your ios app:</p>
+            <div class="frame">
+              <code class="js" data-trim data-line-numbers>
+                "ios": "react-native run-ios”
+              </code>
+            </div>
+
+            <p>Starts Metro Bundler:</p>
+            <div class="frame">
+              <code class="js" data-trim data-line-numbers>
+                "start": "react-native start”
+              </code>
+            </div>
+
+            <p>Example:</p>
+            <div class="frame">
+              <code class="js" data-trim data-line-numbers>
+                "install:full": "rm -rf node_modules && npm install && rm -rf /tmp/metro-bundler-cache-* && cd ./ios && pod install"
+              </code>
+            </div>
+
+          <aside class="notes">
+            •The "scripts" property of your package.json file supports a number of built-in scripts and their preset life cycle events as well as arbitrary scripts. These all can be executed by running npm run <script_name>.
+              Let's see what's inside the default scripts.
+              •npx react-native run-android is one way to run your app - you can also run it directly from within Android Studio.
+              •npx react-native run-ins command will automatically run your app on the iOS Simulator by default.
+              •npx react-native start – starts Metro Bundler
+              
+              Also you can add your own scripts, here is an example:
+              •"install:full": "rm -rf node_modules && npm install && rm -rf /tmp/metro-bundler-cache-* && cd ./ios && pod install"
+          </aside>
+        </section>
+
+        <section class="picture-slide">
+          <h4 class="slide-name">Project structure recommendations</h4>
+          <img src="assets/images/project_st_rec.png" alt="simple" />
```

Comment:
Could you also recommend how to organise assets folder

Reply:
I added some sentences. But if you have smth to add, please let me know. I just not sure what else can we say here